msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: Conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Preapproval of Resource-Specific Consent (RSC) permission allows admins with detailed control over the RSC permissions an app can request during installation. App receives RSC permissions only at the time of installation. However, with preapproval policies, admins can predetermine the maximum permissions an app can request from the end user and obtain consent during installation. 
+
+To enable the preapproval of RSC permission policy for an app, admins must ensure the app can be installed in their organization. For more information, see [managing apps in Teams Admin Center](/microsoftteams/apps-in-teams). If the app can't be installed, you can't enable the preapproval of RSC permission for that specific app.
```

Comment:
Should we rephrase it like this? @AkJo 
"To preapprove RSC permissions for an app"

Reply:
adjusted language for clarity